deployment/terraform/variables.tf

```diff
@@ -283,6 +283,10 @@ variable "app_cc_port" {
   default = "80"
 }
 
+variable "logstash_port" {
```

Comment:
I think it would be more precise if the name is `opensearch_port` because it is basically the OpenSearch port, not Logstash.